src/formatting/generated.rs

```diff
@@ -1,7 +1,7 @@
 /// Returns `true` if the given span is a part of generated files.
-pub(super) fn is_generated_file(original_snippet: &str) -> bool {
+pub(super) fn is_generated_file(original_snippet: &str, header_size: usize) -> bool {
     original_snippet
         .lines()
-        .take(5) // looking for marker only in the beginning of the file
+        .take(header_size) // looking for marker only in the beginning of the file
```

Comment:
please rename `header_size` to something more appropriate or consider passing in `config: &Config` and calling `take(config.generated_marker_line_search_limit())`.

Reply:
Yeah that makes total sense; I've passed the `Config` in. Seems more senisble to me.